gef-ui/templates/index.njk

```diff
@@ -40,9 +40,9 @@
     {% endif %}
 
     {% include "includes/header.njk" %}
+    <div class="govuk-width-container">
```

Comment:
Does this not cut off the navigation bar? It uses it's own comtainer width I believe?

Reply:
When I tried it - visually it spanned the whole width of the screen - this seemed to fix it.